### PR TITLE
Setup backend storage (s3-like) for web app

### PR DESCRIPTION
**Pull request by audiodude**
This PR creates sample entries in credentials.py.example for Development and Production for the s3 storage backend. It also provides a way to get the storage from the web frontend, as well as a way to mock that storage backend in web tests.

### CODE REVIEW

**Comment by kelson42** [comment]
@rgaudin Maybe you can have a look to this PR to see if you don't find any oddities in the module you have created is used,

**Comment by rgaudin** [comment]
> @rgaudin Maybe you can have a look to this PR to see if you don't find any oddities in the module you have created is used,

Looks good

File: wp1/web/storage_test.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                  'key': 'test_key',
+                  'secret': 'test_secret',
+                  'bucket': 'org-kiwix-dev-wp1',
+              }
+          }
+      })
+  @patch('wp1.web.storage.ENV', Environment.DEVELOPMENT)
+  @patch('wp1.web.storage.KiwixStorage')
+  def test_get_storage_sets_storage(self, patched_kiwixstorage):
+    with self.app.app_context() as ctx:
```
Comment (mahakporwal02) [inline wp1/web/storage_test.py:114]: I am not sure if there is a need of `as ctx`.

File: wp1/web/storage_test.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                  'bucket': 'org-kiwix-dev-wp1',
+              }
+          }
+      })
+  @patch('wp1.web.storage.ENV', Environment.DEVELOPMENT)
+  @patch('wp1.web.storage.KiwixStorage')
+  def test_get_storage_returns_s3_from_kiwixstorage(self, patched_kiwixstorage):
+    s3_mock = MagicMock()
+    patched_kiwixstorage.return_value = s3_mock
+    with self.app.app_context() as ctx:
```
Comment (mahakporwal02) [inline wp1/web/storage_test.py:133]: Same

File: wp1/web/storage_test.py [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    with self.override_db(self.app), self.app.app_context():
+      self.assertTrue(has_storage())
+
+  def test_does_not_connect_if_no_creds(self):
+    with self.app.app_context():
+      with self.assertRaises(ValueError):
+        actual = get_storage()
+
+  @patch('wp1.web.storage.KiwixStorage')
+  def test_get_storage_does_not_connect_if_existing(self, patched_kiwixstorage):
```
Comment (mahakporwal02) [inline wp1/web/storage_test.py:26]: Don't you think the name needs to be edited?

**Comment by mahakporwal02** [comment]
LGTM

File: wp1/web/storage_test.py (hunk 3 above)
Comment (audiodude) [reply wp1/web/storage_test.py:26]: I think it's fine.

**Pull request merged**